pkg/engine/templates_generated.go

```diff
@@ -37312,7 +37312,7 @@ installMoby() {
         if [[ "${MOBY_CLI}" == "3.0.4" ]]; then
             MOBY_CLI="3.0.3"
         fi
-        apt_get_install 20 30 120 moby-engine=${MOBY_VERSION} moby-cli=${MOBY_CLI} --allow-downgrades || exit $ERR_MOBY_INSTALL_TIMEOUT
+        apt_get_install 20 30 120 moby-engine=${MOBY_VERSION}* moby-cli=${MOBY_CLI}* --allow-downgrades || exit $ERR_MOBY_INSTALL_TIMEOUT
```

Comment:
PTAL.
The glob here matches the latest version matching the glob.
So, for instance, if we happen to have something like `3.0.10+azure-1` and `3.0.10+azure-2` available, a value of `3.0.10` in the spec will get the `-2`.